deps/jerry/tests/unit-ext/test-ext-handle-scope-escape.c

```diff
@@ -0,0 +1,77 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Unit test for jerry-ext/handle-scope.
+ */
+
+#include "jerryscript.h"
+#include "jerryscript-ext/handle-scope.h"
+#include "test-common.h"
+
+static int native_free_cb_call_count;
+
+static void
+native_free_cb (void *native_p)
+{
+  ++native_free_cb_call_count;
+  (void) native_p;
+} /* native_free_cb */
+
+static const jerry_object_native_info_t native_info =
+{
+  .free_cb = native_free_cb,
+};
+
+static jerry_value_t
+create_object (void)
+{
+  jerryx_handle_scope scope;
+  jerryx_open_handle_scope (&scope);
+  jerry_value_t obj = jerryx_handle_add (jerry_create_object ());
```

Comment:
Make some macros to simplify this function call?

Reply:
🤯Sadly I didn't come up with any possible macro patterns that can be used to add a statement after a declaration. 
Do you have any idea of such kind of macro pattern?